README.md

```diff
@@ -188,7 +188,7 @@ A collection of awesome browser-side [JavaScript](https://developer.mozilla.org/
 * [ESLint](https://github.com/eslint/eslint) - A fully pluggable tool for identifying and reporting on patterns in JavaScript.
 * [JSLint](https://github.com/douglascrockford/JSLint) - High-standards, strict & opinionated code quality tool, aiming to keep only good parts of the language.
 * [JavaScript Standard Style](https://github.com/feross/standard) - Opinionated, no-configuration style guide, style checker, and formatter
-* [Pre-evaluate code at buildtime](https://github.com/kentcdodds/preval.macro) - Pre-evaluate your front end javascript code at build-time
+* [Pre-evaluate code at build time](https://github.com/kentcdodds/preval.macro) - Pre-evaluate your front end javascript code at build-time
```

Comment:
Replacing space with a `-` might be an option, since the description is also like this:
> Pre-evaluate code at build-time with babel-macros